libcasr/src/asan.rs

```diff
@@ -173,7 +173,9 @@ impl Severity for AsanContext {
         if asan_report[0].contains("LeakSanitizer") {
             ExecutionClass::find("memory-leaks")
         } else {
-            let summary = Regex::new(r"SUMMARY: *(AddressSanitizer|libFuzzer): (\S+)").unwrap();
+            let summary =
+                Regex::new(r"SUMMARY: *(AddressSanitizer|libFuzzer): ([0-9A-Za-z_\-\(\)]+)")
```

Comment:
Are digits needed here?

Reply:
I am not sure. Can there be any UBSAN warnings containing digits? Or can there be something like funcnamewithdigits5-param-overlap?

Reply:
Maybe, also, underscope is redundant...

Reply:
Underscope is not redundant, cuz we have `bad-malloc_usable_size` class

Reply:
As I know, func names and ubsan errors are not relevant here. This code only parse asan execution_class, I guess

Reply:
> As I know, func names and ubsan errors are not relevant here. This code only parse asan execution_class, I guess

However, there is the same regex for UBSAN in this PR.

Reply:
Ubsan error types don't contain digits too
https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/ubsan/ubsan_checks.inc

Reply:
removed digits